tools/windows-node-installer/test/e2e/clustersetup.go

```diff
@@ -0,0 +1,83 @@
+package e2e
+
+import (
+	"fmt"
+	"io/ioutil"
+	"k8s.io/client-go/tools/clientcmd"
+
+	authscheme "github.com/openshift/client-go/authorization/clientset/versioned/scheme"
+	v1auth "k8s.io/api/rbac/v1"
+	k8srbac "k8s.io/client-go/kubernetes/typed/rbac/v1"
+	"k8s.io/client-go/rest"
+)
+
+const (
+	configDir   = "clusterconfig"
+	projectName = "default"
+)
+
+var configFiles = [...]string{"role.yaml", "clusterole.yaml", "cluster_role_binding.yaml", "role_binding.yaml"}
+
+func authConfig(cfg *rest.Config) error {
```

Comment:
Perhaps I'd have done it within a shell script. No need to do it via go code, usually it makes hard to distribute a single go  binary